devices/tuya.js

```diff
@@ -1175,13 +1288,15 @@ module.exports = [
             {modelID: 'TS0202', manufacturerName: '_TZ3000_h4wnrtck'},
             {modelID: 'TS0202', manufacturerName: '_TZ3000_sr0vaafi'},
             {modelID: 'WHD02', manufacturerName: '_TZ3000_hktqahrq'},
+            {modelID: 'TS0202', manufacturerName: '_TZ3040_wqmtjsyk'},
         ],
         model: 'TS0202',
         vendor: 'TuYa',
         description: 'Motion sensor',
         whiteLabel: [{vendor: 'Mercator Ikuü', model: 'SMA02P'},
             {vendor: 'TuYa', model: 'TY-ZPR06'},
-            {vendor: 'Tesla Smart', model: 'TS0202'}],
+            {vendor: 'Tesla Smart', model: 'TS0202'},
+            {vendor: 'MiBoxer', model: 'PIR1-ZB'}],
```

Comment:
Please use new whitelabel here (such that device will be detected correctly): https://github.com/Koenkk/zigbee-herdsman-converters/blob/e630d74e36761c806a5e05f321570d2bb486bee9/devices/tuya.js#L1029

Reply:
done